src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -0,0 +1,441 @@
+
+                this.rowTemplates = templates;
+            },
+
+            show: function () {
+                if (this.rows.length > 0) {
+                    completions.selectedRow = undefined;
+                    this.state = this.VISIBLE;
+                    this.update();
+                    completions.element.show();
+                    events.onShow();
+                }
+            },
+
+            hide: function () {
+                if (completions.state === completions.HIDDEN) {
+                    return;
+                }
+                completions.state = completions.HIDDEN;
+
+                window.setTimeout(function () {
+                    /*
+                     * This hack is necessary to allow the click event to be handled before the element is hidden.
+                     *
+                     * Order of events:
+                     * 1. mouseDown
+                     * 2. focusOut ( will hide the autocompletion)
+                     * 3. click
+                     *
+                     * The focus check is necessary because the input field could already be focused (again).
+                     * In that case the autocomplete should not be hidden.
+                     */
+                    if (completions.state === completions.VISIBLE || input.element.is(":focus")) {
+                        return;
+                    }
+
+                    completions.hideImmediate();
+                }, 500);
+            },
+
+            hideImmediate: function () {
+                if (completions.state === completions.HIDDEN) {
+                    return;
+                }
+                completions.state = completions.HIDDEN;
+
+                completions.element.hide();
+                events.onHide();
+            },
+
+            /**
+             * Retrieves the template for the given type.
+             *
+             * @@param type string which represents the type name
+             * @@returns string representing the template of the given row type
+             */
+            templateForType: function (type) {
+                if (type && Object.keys(this.rowTemplates).includes(type)) {
+                    return this.rowTemplates[type];
+                }
+
+                return this.rowTemplates.basic;
+            },
+
+            /**
+             * Updates the rows in the autocomplete wrapper
+             */
+            update: function () {
+                this.element.empty();
+
+                if (this.rows.length < 1){
+                    completions.hideImmediate();
+                    return;
+                }
+
+                $.each(this.rows, function (index, row) {
+                    var template = completions.templateForType(row.type);
+
+                    var $row = $(Mustache.render(template, row));
+                    completions.addHoverRowEvent($row);
+                    completions.element.append($row);
+                });
```

Comment:
after rendering, we need to call lazyload on the images inside the dropdown. I see no event or other possiblity to take this action after every update

Reply:
-> events.afterRender() & events.afterRenderRow()